cosmos/providers/databricks/notebook.py

```diff
@@ -94,7 +96,7 @@ def convert_to_databricks_workflow_task(
         Convert the operator to a Databricks workflow task that can be task in a workflow
         """
         result = {
-            "task_key": self.dag_id + "__" + self.task_id.replace(".", "__"),
+            "task_key": self._get_databricks_task_id(),
             "depends_on": [
                 {"task_key": self.dag_id + "__" + t.replace(".", "__")}
```

Comment:
We should have a unified function to process `_get_databricks_task_id` and this line - otherwise, if the logic changes, we can have issues